src/ResourceManager/Resources/Commands.Resources/Models.ResourceGroups/ResourceClient.cs

```diff
@@ -313,16 +313,28 @@ public static List<string> ParseDetailErrorMessage(string statusMessage)
         {
             DeploymentExtended deployment;
 
+            // Poll deployment state and deployment operations.
```

Comment:
Thanks for catching, I I thought i missed that change in this PR and opened a new one. I am closing the other one